plugins/modules/mount.py

```diff
@@ -734,105 +735,116 @@ def main():
     name = module.params['path']
     changed = False
 
-    if state == 'absent':
-        name, changed = unset_mount(module, args)
+    lock_timeout = 2
```

Comment:
Do you think we should make it configurable? Just a thought.

Reply:
maybe later, if someone reports it's not enough ? In my tests, with 24 *samehosts* in parallel, there was no error reported for 100 runs of the same play, while the same tests without locking fstab report >200 errors.